libfreerdp/freerdp.pc.in

```diff
@@ -3,6 +3,11 @@ exec_prefix=@CMAKE_INSTALL_PREFIX@
 libdir=@CMAKE_INSTALL_FULL_LIBDIR@
 includedir=${prefix}/@FREERDP_INCLUDE_DIR@
 libs=-lfreerdp@FREERDP_API_VERSION@
+datarootdir=${prefix}/share
```

Comment:
This exposes a directory in the pkgconfig, that also contains non-freerdp related stuff of other packages. According to the commit message, this doesn't look to be intended, or not?

Reply:
@pnowack i't the common way used by many packages to have a variable for the base path for data files. Basically the same as libdir

Reply:
@pnowack this is the `datarootdir`, so should be ok. (grep in `/usr/lib64/pkgconfig`  on fedora shows all packages set it like this)

Reply:
> (grep in /usr/lib64/pkgconfig on fedora shows all packages set it like this)

Oh, indeed. Let's keep it then.